sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/tests/TimeSeriesInsightsQueryAggregateSeriesTests.cs

```diff
@@ -154,10 +154,10 @@ await foreach (Page<TimeSeriesPoint> aggregateSeriesPage in queryAggregateSeries
                 sendEventAct.Should().NotThrow();
 
                 // Query for the two events with a filter
-                queryAggregateSeriesRequestOptions.Filter = "$event.Temperature.Double = 1.2";
-                await TestRetryHelper.RetryAsync<AsyncPageable<QueryResultPage>>(async () =>
+                queryAggregateSeriesRequestOptions.Filter = new TimeSeriesExpression("$event.Temperature.Double = 1.2");
```

Comment:
same here

Reply:
I wonder if GH is confused on what changed in this line. All I did was construct a TimeSeriesExpression with the string.